geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloRunner.scala

```diff
@@ -93,24 +98,42 @@ object AccumuloRunner extends Runner {
       if (p.zookeepers == null) {
         p.zookeepers = zookeepers
       }
-      if (p.instance == null) {
-        p.instance = try {
-          // This block checks for the same system property which Accumulo uses for Zookeeper timeouts.
-          //  If it is set, we use it.  Otherwise, a timeout of 5 seconds is used.
-          //  Don't give default to GeoMesaSystemProperty so .option will throw a None
-          val lookupTime: Long = SystemProperty("instance.zookeeper.timeout").option.flatMap{ p =>
-            Try { java.lang.Long.parseLong(p) }.toOption
-          }.getOrElse(5000L)
+      try {
+        // This block checks for the same system property which Accumulo uses for Zookeeper timeouts.
+        //  If it is set, we use it.  Otherwise, a timeout of 5 seconds is used.
+        //  Don't give default to GeoMesaSystemProperty so .option will throw a None
+        val lookupTime: Long = SystemProperty("instance.zookeeper.timeout").option.flatMap{ p =>
+          Try { java.lang.Long.parseLong(p) }.toOption
+        }.getOrElse(5000L)
 
-          Command.user.debug(s"Looking up Accumulo Instance Id in Zookeeper for $lookupTime milliseconds.")
-          Command.user.debug("You can specify the Instance Id via the command line or\n" +
-            "change the Zookeeper timeout by setting the system property 'instance.zookeeper.timeout'.")
+        Command.user.debug(s"Looking up Accumulo Instance Id in Zookeeper for $lookupTime milliseconds.")
+        Command.user.debug("You can specify the Instance Id via the command line or\n" +
+          "change the Zookeeper timeout by setting the system property 'instance.zookeeper.timeout'.")
 
-          import scala.concurrent.duration._
-          Await.result(Future(HdfsZooInstance.getInstance().getInstanceName)(ExecutionContext.global),  lookupTime.millis)
+        import scala.concurrent.duration._
+        lazy val ff: Either[Throwable, String] = try {
+          // This call not only provides the instance name but in situations where the instance
+          // name is provided, this allows us to check that we have access to Accumulo dependencies.
+          val i = HdfsZooInstance.getInstance().getInstanceName
+          Right(i)
         } catch {
-          case NonFatal(e) => Command.user.warn(s"Exception getting zoo instance: ${e.toString}"); null
+          case e: Throwable => Left(e)
         }
+        val future = Await.result(Future(ff)(ExecutionContext.global),  lookupTime.millis)
+
+        future match {
+          case Right(instanceId) => if (p.instance == null) p.instance = instanceId
```

Comment:
Added comments. 
I changed the check to look at the class of the params for the command that was invoked. This should mostly prevent unnecessary checks and allow commands that don't use Accumulo resources through. 
Added check and warning if the instance names don't match. I don't really understand the implications/repercussions here so that seems appropriate.